fluent-plugin-events/test/helper.rb

```diff
@@ -12,9 +13,24 @@ def test_resource(name)
   File.new("test/resources/#{name}")
 end
 
-def mock_get_events
+def stub_apis
```

Comment:
can you explain how this works?

Reply:
The reason I added this because the unit test doesn't pass before. The reason is basically that the clients cannot be created successfully. I try to print out the object `@clients`. It looks like following:
```
clients {"v1"=>"2019-07-16 21:15:40 +0000 [error]: Failed to open TCP connection to localhost:8080 (Connection refused - connect(2) for \"localhost\" port 8080)", "events.k8s.io/v1beta1"=>"2019-07-16 21:15:40 +0000 [error]: Failed to open TCP connection to localhost:8080 (Connection refused - connect(2) for \"localhost\" port 8080)", "apps/v1"=>"2019-07-16 21:15:40 +0000 [error]: Failed to open TCP connection to localhost:8080 (Connection refused - connect(2) for \"localhost\" port 8080)", "extensions/v1beta1"=>"2019-07-16 21:15:40 +0000 [error]: Failed to open TCP connection to localhost:8080 (Connection refused - connect(2) for \"localhost\" port 8080)"}, clients type Hash
```
It will get the error of connection issues when creating clients. I think the reason why it won't cause problem locally is that I have `kubectl proxy --port=8080` running locally. So I added this to make sure there won't have this TCP connection issues.

Reply:
hmm okay, I still don't know _how_ it fixes the issue? can you explain what `stub_apis` does?

Reply:
What `stub_apis` does here is to mock response for `client.api_valid?`. Otherwise, method `create_client` will be stuck here with the connection error as I mentioned before.

Reply:
ah interesting. thanks for clarifying